[1.1.x] Add `apiArtifactsMountEmptyDir` and `dropinsMountEmptyDir`configs (#59)
wso2/kubernetes-microgateway: helm/choreo-connect/README.md

| `wso2.deployment.adapter.podAnnotations`                                    | Key value pair of annotations for the pod                                                 | sidecar.istio.io/inject: "false" |
| `wso2.deployment.adapter.configToml`                                        | Define templated config.toml file, if empty using default config.toml                     | Default templated config toml file |
| `wso2.deployment.adapter.logConfigToml`                                     | Define templated log_config.toml file, if empty using default log_config.toml             | Default templated log config toml file |

> | `wso2.deployment.adapter.apiArtifactsMountEmptyDir`         | Mount API artifacts directory if the directory is empty         |           false         |

Reviewer: ```suggestion
| `wso2.deployment.adapter.apiArtifactsMountEmptyDir`         | Mount an empty directory on API artifacts directory         |           false         |
```

| `wso2.deployment.gatewayRuntime.enforcer.imagePullPolicy`                   | Image pull policy of the container                                                        | "IfNotPresent"              |
| `wso2.deployment.gatewayRuntime.enforcer.envOverride`                       | Set (or override) environment variables as values or from ConfigMaps or Secrets           | Default Passwords           |
| `wso2.deployment.gatewayRuntime.enforcer.dropins`                           | Mount enforcer lib dropins JARs to the `dropins` directory, array of ConfigMap names      | -                           |

> | `wso2.deployment.gatewayRuntime.enforcer.dropinsMountEmptyDir`          | Mount dropins JARs directory if the directory is empty          |           true          |

Reviewer: ```suggestion
| `wso2.deployment.gatewayRuntime.enforcer.dropinsMountEmptyDir`          | Mount an empty directory on dropins JARs directory          |           true          |
```
